test/index.js

```diff
@@ -4,18 +4,11 @@
 require('source-map-support').install();
 
 global.sinon = require('sinon');
+require('./chai-setup');
+global.chai.use(require('sinon-chai'));
```

Comment:
Perhaps this should be moved into `test/chai-setup.js` as well?

Reply:
Nope, because this related to sinon too

Reply:
It's only "related to sinon" in the sense that it delegates to the sinon library under the hood.  However, it does not actually change the behavior of sinon itself in any way.  (It's a plugin that adds a handful of new functions to chai)

But yeah -- it's not really important for this PR anyway, so we can skip this change for now.